p2p/src/net/libp2p/types.rs

```diff
@@ -159,9 +159,6 @@ pub enum ConnectivityEvent {
     /// One or more peers that were previously discovered have expired
     Expired { peers: Vec<(PeerId, Multiaddr)> },
 
```

Comment:
So now there's no event for when a peer gets disconnected? Or are we gonna use the same ConnectionClosed for that?

Reply:
`ConnectionClosed`